services/web/src/components/RequestBlock/index.js

```diff
@@ -0,0 +1,68 @@
+import { Layout } from 'components';
+import React from 'react';
+import { Button, Header, Dropdown } from 'semantic';
+import CodeBlock from '../CodeBlock';
+import templateCurl from './templates/curl';
+import templateFetch from './templates/fetch';
+
+const options = [
```

Comment:
prefer caps casing for constants to lower case

forgive me I'm going super nitty on this one

Reply:
hehe this one i have hard time to eat ... 
I would agree if it was like a env var or api-key sure.

Reply:
ok not hard on it but it's the convention I always use

Reply:
can you outline the benefit, and i could try to shallow it, in the name of having one style. 
the only benefit then you can avoid some https://eslint.org/docs/rules/no-shadow. (which i think we should enable, but i seem to recall some objection for it)

If we do eat it ... then let's make sure its also enforced ... if its not enforced its doesn't matter.

Reply:
I'm not sure we're talking about the same thing. I just prefer using all caps for constants as it makes them easy to scan in large files and the intent clear. "shadowing" shouldn't have anything to do with it I think?